dxr/lines.py

```diff
@@ -35,6 +36,133 @@ def __repr__(self):
 LINE = Line()
 
 
+class Ref(object):
+    """A cross-reference attached to a run of text
+
+    Carries with it a context menu and other metadata.
+
+    """
+    sort_order = 1
+    __slots__ = ['menu', 'hover', 'qualname_hash']
+
+    def __init__(self, menu, hover=None, qualname=None, qualname_hash=None):
+        """
+        :arg menu: a list of MenuMakers that generate the context menu
```

Comment:
In the spirit of Name it What You Call it, would it make more sense to name this arg to menu_makers?